CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+# 84.6.0
+
+In this minor release we added new features to ActionSheet and NavigationDrawer, as well as additional tests and bug fixes.
+
+## New features
+
+ActionSheet now allows clients to align all titles regardless of if an image is present or not.
+
+```swift
+let actionSheet = MDCActionSheetController()
+actionSheet.alwaysAlignTitleLeadingEdges = true
+```
+
+NavigationDrawer now allows clients to include the safeAreaInsets to the content height.
```

Comment:
Tip: Suggestion: "...to automatically include `safeAreaInsets` in the content height calculation."

Reply:
Done